types/wordpress/blocks.d.ts

```diff
@@ -65,7 +67,7 @@ export interface GutenbergAntDesignButtonBlockProps extends CoreBlockProps {
     disabled?: boolean;
     ghost?: boolean;
     href?: string;
-    htmlType?: string;
+    htmlType?: "submit"| "button" | "reset";
```

Comment:
@Danishkhurshid This shouldn't be limited, can potentially be one of many different types that HTML provides for different elements